spec/board_rspec.rb

```diff
@@ -0,0 +1,91 @@
+require './lib/board'
+require './bin/main'
```

Comment:
- **[SUGGESTION]** - I was unable to run the tests until I removed the game initialization lines from main.rb file.  As a convention for this project, all classes should be kept in /lib folder. I suggest you add a file for your UI class in the /lib directory and import it from main.rb so you don't have to import the latter in your specs.

Reply:
Same as Above, file were missing added , you dont need to do any changes of removing game init so far.